docs/security/simple-ldap-variables.md

```diff
@@ -1,4 +1,28 @@
-# Simple LDAP Variables
+# Simple LDAP authentication variables
+
+## Variables installed
+
+The installation adds the following variables:
+
+| Variable name | Description |
+|---|---|
+| authentication_ldap_simple_bind_base_dn | Base distinguished name |
+| authentication_ldap_simple_bind_root_dn | Root distinguished name |
+| authentication_ldap_simple_bind_root_dn_pwd | Password for the root distinguished name |
+| authentication_ldap_simple_ca_path | Absolute path of the certificate authority |
+| authentication_ldap_simple_group_role_mapping | LDAP group name |
+| authentication_ldap_simple_group_search_attr | Name of the attribute that specifies the group names in the LDAP directory entries |
+| authentication_ldap_simple_group_search_filter | Custom group search filter |
+| authentication_ldap_simple_init_pool_size | Initial size of the connection pool to the LDAP server |
+| authentication_ldap_simple_log_status | logging level |
+| authentication_ldap_simple_max_pool_size | Maximum size of the pool of connections to the LDAP server |
+| authentication_ldap_simple_server_host | LDAP server host |
+| authentication_ldap_simple_server_port | LDAP server TCP/IP port number |
```

Comment:
The fallback variables aren't mentioned.

Reply:
Added